examples/custom_widget_example.py

```diff
@@ -130,7 +130,9 @@ def draw(self, obj, draw_ctx):
 class CustomWidget():
 
     # An instance of a widget-class to be used for creating custom widgets
-    cls = CustomWidgetClass(lv.DPI.DEF, lv.DPI.DEF)
+    d = lv.disp_get_default();
```

Comment:
Minor nitpick, but the semicolon is unnecessary here, right?

Reply:
Right.  
Actually there were lots of spurious semicolons...  
I now removed those I found.